### ISSUE TITLE
[Design] My VA: Convert mobile wireframes to desktop layout

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#17307]
## Background

In #16807, we finished up the wireframe layout for the mobile view. Those need to be converted to a desktop layout so that can be finalized and we can start frontend build.

A few notes:

- We are *not* going to have a side navigation on desktop at this point in time, so please do not include one.
- Please try to avoid a single-column layout. It's one of the major problems we have now with the page that makes it impossible to scale.
- We may discover some things that work on mobile don't work when we convert to desktop, so it's OK if we discuss what changes might need to happen to make the design work for both desktop and mobile. That means that at the end of this, we may need to go back and tweak the mobile wireframe.
- Please do not add data points or elements on the desktop view that are not on the mobile view **unless** the plan is to go back and update the mobile view. The two views shouldn't have different data or different elements unless there's a very good reason.

And some notes on process:

- I'll leave it up to you whether you'd like to handle all sections at once or whether it is easier to break this down like we did with the mobile view and work on one section at a time.
- I expect we'll be in daily communication about this like we were with the mobile wireframes.

## Tasks

- [ ] Convert mobile wireframe for My VA to desktop layout

**Comment by callen2563** [comment]
@Samara-Strauss @tressaellen 
This is a first pass at the desktop layout, we aren't quite ready for feed back yet as the the claims container is too squished so we are working on another version to treat disability differently. Also we changed the links to buttons because they were getting lost on desktop as well as to comply with the style guide and accessibility. We will update mobile as well. However, we wanted to post our progress on desktop so far. The next version is on its way. Thanks! :)

![myVA desktop Traditional Dashboard V1](https://user-images.githubusercontent.com/68560244/102121651-26adaf80-3df9-11eb-83f7-37b101dfce34.jpg)

**Comment by callen2563** [comment]
@Samara-Strauss @tressaellen 
This is a first pass at the desktop layout, the claims container is too squished so we created another couple versions to treat disability differently. Also we changed the links to buttons because they were getting lost on desktop as well as to comply with the style guide and accessibility. We will update mobile as well. Let us know your thoughts. Thanks! :)


![myVA desktop Traditional Dashboard V1](https://user-images.githubusercontent.com/68560244/102125440-907c8800-3dfe-11eb-89a3-0b6dbc890483.jpg)
![myVA desktop Traditional Dashboard V3](https://user-images.githubusercontent.com/68560244/102125455-95413c00-3dfe-11eb-9df3-1a23673493b8.jpg)
![myVA desktop Traditional Dashboard V2](https://user-images.githubusercontent.com/68560244/102125428-8bb7d400-3dfe-11eb-9a52-fa0ec1e9a421.jpg)

**Comment by Samara-Strauss** [comment]
Adding notes from our 12.14.20 chat:

- We are going to move into visual design for the desktop view, because that will make it much easier to determine what actually works in terms of layout and info hierarchy.
- Like before, we'll probably check in every day.
- The goal is to have a layout and information hierarchy we feel comfortable with by the end of the week (if possible). The visual elements do not have to be perfect, though they shouldn't be intentionally sloppy, and they can change and be refined after break.
- It is highly encouraged that if you think we need to update some of the data or hierarchy that you share those recommendations. As long as those recommendations are founded in good UX thinking, user needs, and data we actually have access to, then please share so we can discuss.
  - Some potential examples — showing the most recent claim instead of the most recent three; disability rating should be shown in a separate and distinct way away from the updates, etc etc. I am NOT saying we should/should not do any of these, but these are the types of things you might suggest, and I am totally open to them if your reasoning is strong!

Most importantly, **this is not about guessing what Samara likes/wants/what's in her head**. I am **not** the user — veterans are! The suggestions I make are me trying my best to figure out how to serve veterans and [what they need this page to do](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2.0-redesign/product/my-va-jobs.md). But I don't always get it right, and that's why this work needs to be more of a conversation and less me imparting things from on high. Please feel free to throw out any layout suggestions I made with the gray boxes from a couple weeks ago. Those were only to communicate **one** idea and to communicate hierarchy. But there are LOTS of ways this page could shake out, and I am looking to you all to lead on how we can make this match user expectations and needs.

Let me know if you have questions! Looking forward to chatting tomorrow!

**Comment by Samara-Strauss** [comment]
Making a note that we may not finish this in sprint 37 due to holiday time off, but hopefully we can make good progress!

**Comment by Samara-Strauss** [comment]
Wanted to add a note about what I mentioned in standup:

I think we should play with when and how we lean more towards showing just the status of something vs. leaning more towards showing the actual, full update details for something. Here are some non-prescriptive examples of how you might do that for a couple of modules:

### Example 1: Appointments

**Super status-y**

For something that leaned heavily towards status but not updates, we might show the following stats:

- 1 appointment this week
- 4 upcoming appointments
- **1 canceled appointment**

[View appointments]()

**Some status, some "update"**

For a middle ground, we might show

- A block showing full details for your next upcoming appointment

(And then below that)

You also have:

- 4 more upcoming appointments
- **1 canceled appointment**

[View appointments]()

**Super update-y**

In something that leaned heavily towards full update details, we might show the full details for your next 3 upcoming appointments, plus a CTA to view appointments. We might also show some sort of messaging if an appointment has been canceled.

### Example 2: Claims

**Super status-y**

For something that leaned heavily towards status but not updates, we might show the following stats:

- 1 claim has recently been updated [View claim]()
- 2 total claims open

[View all claims]()

**Some status, some "update"**

Here is your most recent claim or appeal update:

(show the update here)

You also have:

- 2 more claims or appeals that have been updated recently
- 5 total claims open

[View all claims]()

**Super update-y**

In something that leaned heavily towards updates, we might show the full details for their most recent claim or appeals updates, with up to 3 claims able to show.

----

Since this has caused confusion in the past, **I am not prescribing anything here.** The goal of this is to just give some ideas about how we can balance and prioritize information, and to hopefully give you some things to consider as you're doing that. I think @tressaellen brought up a really good point when I mentioned this idea of "status vs. update" in standup, but there might be some blocks for which you lean more towards status (eg. maybe it's enough to just show "2 unread messages" or "2 prescriptions refills submitted; 1 prescription recently filled") instead of showing the full update details, and maybe for some you lean more heavily towards showing full update details (eg. showing full appointment details for at least 1 appointment; showing full details for claims updates). **And that is ok**. We might not need to treat everything with exactly the same weight.

Please let me know if you have thoughts or questions about this. I am happy to clarify anything that is confusing!

**Comment by tressaellen** [comment]
Hello @Samara-Strauss (and @rtwell, @sshein )! (cc/@callen )

We'd like to schedule a call with you to walk you through what we have - but we wanted to give you a change to look over things before we chat. If you can meet in the morning, that would be preferable, to make sure we are all on the same page. A few thoughts about the screens below:

These are not meant to be visual designs yet - we realize they are 'high fidelity wireframes' (if you will), but we have not started on visuals yet. This is more to nail down the hierarchy, layout, and appropriate level of information.

1-5 are exploring hierarchy and level of information (ie including more details on prescriptions and appointments).  @callen2563 and I prefer #2 - we think that it most efficiently meets the goals that we found in research as well as the goals that were outlined for MyVA (listed below)

- As a veteran, I want to be able to log in to VA.gov and easily find the tools and tasks that apply to my specific experience at the VA.
- As a veteran, I want to be able to log in to VA.gov and easily see status updates and notifications that apply to my specific experience at the VA.
- As a veteran, I want to be able to log in to VA.gov, quickly complete whatever I came to do, and then get back to the rest of my life. 

6-7 are exploring what you outlined in the comment above (status-ey vs update-ey) - I think this has some interesting benefits... I also think that we need to explore this a little bit more if we want to go in this direction.

We are definitely open to feedback and whatever questions you have.  Let us know what time works for you to chat, we're excited to hear your thoughts.

![MyVA 1.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/e3b41afa-09b3-4708-92d8-b960710832b1)
![My VA 2.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/0b2fe1f9-ac25-4ac7-b73e-ec3f95c2fdfa)![My VA 3.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/983255e5-0f75-49bc-99f2-8dfe87defc7d)![My VA 4.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/61b22179-d4fe-4ec6-aa8f-f6d7a9630c6c)![My VA 5.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/c7c15e23-44b1-4d73-9977-dc41aee12535)!


[MyVA 6.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/dbd10781-6600-4696-ba60-e80443f6dcf5)![MyVA 7.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/f0167cf5-de34-48af-8fad-c7ec3cc5039e)

**Comment by tressaellen** [comment]
Also, for documentations sake, these are the cards (aka lego blocks) that we have for each of the sections.

![myVA card status.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/2a9aea12-148c-407c-892a-59fcc21ca17f)
![myVA card status blank.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/2b42f2e0-e4e0-4bd7-82cf-5b8b1714d263)

**Comment by Samara-Strauss** [comment]
Thanks for this! Let's definitely regroup early today. Let me know when you're in! In the meantime, a few preliminary thoughts:

**Information hierarchy**

I understand that in research, we heard that people wanted to have the most actionable items at the top, but the groupings of information in option 2 don't really make sense to me. I totally understand these aren't really communicating layout, and I am not sure if those groupings were intentional or not, BUT it appears they might be? And in that case, I'm not sure why you grouped disability rating with messages and prescriptions.

In terms of hierarchy and grouping options that made the most sense to me, options 1, 3, or 4 felt the most natural. This is because I think there is a natural association between appointments, messages, and prescriptions since they are all health care related. And yes, I realize this puts us in a position where you have to choose to prioritize a claim update over a new message or vice versa, but that's OK. We also don't want to ignore that certain types of information are naturally going to be associated (and I think our participants reflected this in user research).

Also, I think we could go either way on associating disability rating and claim status. Associating them likely makes sense in the veteran's mental model (remember, veterans sometimes will check to see if their disability rating has changed as a proxy metric for whether a claim was approved), BUT there is the very fundamental difference between one is a status and one is an update, so some separation might be OK.

**Status vs. update — how much do we show for each type of information?**

- I think it's totally ok to go more update-y with some things and more status-y with others. Personally, I thought option 7 had the right level of priority — more info for claims and appointments, less for disability rating, messages, and prescriptions. BUT, I'm hoping the grouping in option 7 was NOT intentional. I don't think it makes any sense to associate messages and prescriptions with disability rating, nor to relegate the stuff to a side rail. I still think messages and prescriptions should be more closely grouped with appointments, even if there is a call out for a patient's next appointment.

Let's chat soon!

**Comment by tressaellen** [comment]
We met with @Samara-Strauss and landed on this. We still have some details to figure out around prescriptions. Next steps will be to design for each state of each section (someone with no claims, someone with no upcoming appointments etc.)

![MyVA Frankenstein.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/5d1bb9e0-2e95-47a2-9189-12379823a8f2)

**Comment by callen2563** [comment]
@Samara-Strauss @rtwell @sshein @tressaellen 

Hi Samara,
Tressa and I just finished up defining the use cases of the status updates. We kept the prescription section simplistic for V1, some of the details are still subject to change in that section. Next steps when we get back from break will be to edit these and evaluate how to add in the additional links that you mentioned in slack. From there we will move onto a final wire frame for mobile and desktop. Hope you have a wonderful holiday and excited to hit the ground running when we return! 

![MyVA use cases : status updates.jpg](https://images.zenhubusercontent.com/5f1aec91bccc130a8ff98161/28f8e478-d1c3-4090-bbef-058a72f8473c)

**Comment by Samara-Strauss** [comment]
Thanks for this! I think this is most of the way there, but some tweaks are needed.

### Claims & appeals status

**A user who has multiple claims in progress**

- Is the CTA going to be "View 3 more updates?" If so, I think this could be confusing to users because it is unclear what "updates" means. And remember, regardless of what the updates are, we are going to link to the same place for all of them — the claim status tool.

**A user who has multiple claims but no other updates**

- Should have a View all claims CTA.

**A first time user who has never filed a claim**

- In this case, I was thinking we would NOT show the claim & appeals status section, and that the prompt to file a claim for the first time would live in the "Applying for benefits" section.

### Appointments

**A user who has multiple appointment updates**

- Same comment as with claim status — "View 3 more updates" is confusing, though Idk if that's actually what you intend the CTA to be.

**A user who has multiple appointments but no other updates**

- Should have a CTA to View all appointments or Manage appointments.

**A user who has 1 total upcoming appointment**

- Is the "1 total upcoming appointment" the next appointment? If so, I do not think we should show "1 total upcoming appointment" in addition to the next appointment. It's confusing and makes me think I have 2 appointments (the next appointment, and one more after that). 
- Should have a CTA to View all appointments or Manage appointments.

**A first time user who doesn't have any health care**

- We will not show the health care block at all if someone isn't enrolled in health care, so this status will not exist. We can show a prompt for people to apply for healthcare for the first time in the "Applying for benefits" section.

### Prescriptions

**A user who has multiple prescription updates**

- Same comment as above — "Updates" needs to be clarified.
- This should have a CTA to View or Manage all prescriptions.

**Most recent prescription update**

- This should have a CTA to View or Manage all prescriptions.

**A user who as 1 prescription update**

- I don't think we need "1 total prescription" if the person only has 1.
- This should have a CTA to View or Manage all prescriptions.

**A user who has no current prescriptions in progress, but DOES have prescriptions**

- There should be some sort of clarifier copy here in addition to "fill a prescription."

**A user who has health care but does not have any prescriptions at the VA**

- This use case was not accounted for. In this case, I think we can just not show the prescription section at all.

**A first time user who doesn't have any health care**

- We will not show the health care block at all if someone isn't enrolled in health care, so this status will not exist. We can show a prompt for people to apply for healthcare for the first time in the "Applying for benefits" section.

### Messages

- We need to bold the copy there.
- For "you have 2 new messages", that can likely be copy, with a CTA button/link for "View all messages."

### Applying for benefits

This does need to be defined before moving on, so I wanted to include some thoughts on how I see this working with a very basic first-time user experience (FTUX).

**A user has no benefits at the VA and has never applied for anything**

- Show a CTA prompt to apply for health care.
- Show a CTA prompt to file a claim for the first time.
- Show a CTA prompt to apply for education benefits.

We can also still show the FAQ for "What benefits can I apply for?"

**A user has started an application for [health care, disability, education]**

- Once a user has started an application for health care, disability, or education, we convert the prompt to start that application to the "in-progress" state, if applicable, or we remove the prompt entirely if the application is started and submitted in one sitting.
- **The prompt to start a health care, disability, or education application never comes back after someone has started/submitted an application, even if that application is ultimately rejected.**

**A user has started an application and saved it in progress for something other than health care, disability, or education**

- We'll handle this exactly how we are now — if the app is saved in progress, we show it on the dashboard.

**The TL;DR**

- There's really no change to how we are handling applications — if they are saved in progress, we'll show them in the "Applying for benefits" section. All I am adding is the idea that if someone has never done anything at the VA and has not applied for the three most-used benefits, we show prompts to encourage them to applying for those benefits, and those prompts will be removed as soon as they start an application.

**Comment by Samara-Strauss** [comment]
Item for discussion: Should we add a few other popular links given that they are frequently used on the current page? [Here is the report for top clicks from My VA for the 2020 calendar year](https://analytics.google.com/analytics/web/?authuser=0#/report/content-event-pages/a50123418w177519031p176188361/_u.date00=20200101&_u.date01=20201231&explorer-table.plotKeys=%5B%5D&_r.drilldown=analytics.pagePath:www.va.gov~2Fmy-va~2F,analytics.eventCategory:Interactions/). You can see the following in the top 10:

- View Profile
- Download letters
- Check Post-9/11 GI bill benefits

Can we/should we fit these in? Let's discuss.

**Comment by tressaellen** [comment]
For documentations sake - @Samara-Strauss , @callen2563 and I met this morning to go over state types. What we settled on is below (I will also be putting this in a GH folder).  

![Status Types.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/588b529a-bcb3-4ce0-894a-4c807ab16fb8)

**Comment by Samara-Strauss** [comment]
Can you add that file to [the GitHub folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/my-va/2.0-redesign/design-ia/assets)? That way, it won't get lost to time, which it will if it only lives in this ticket. Thanks!

**Comment by tressaellen** [comment]
@Samara-Strauss - yep I'll add it into GH. 

Also, here are the wireframes for the main use cases - if a user has everything (i.e. healthcare, claims, benefits applications), if a user has no new updates, if a user has logged-in for the first time (FTUX), if a user has healthcare but not claims, and if a user has claims but no healthcare. There are still a lot more use cases that need to be handled, but this is a good jumping off point to start thinking about/handling visual design and layout.

These are **still wireframe blocks**, we just wanted to get a feel for how much information would live on the page for each user. We realize this isn't completely accurate, since we still have placeholder text in each of the 'cards', BUT we thought it would still be valuable.  Also fyi, in these pages we have used a right rail. This is just a placeholder until we move into visuals, but TBD until we move on. 

![My VA User who has everything, right rail.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/c22d64bb-60ca-4dcf-b04c-d8d2f5be3052)

![My VA User who has no new updates.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/e2606779-ed8f-4aa9-8e74-4aa9806778a2)

![MyVA FTUX.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/9b0a3cef-3bcf-4bbb-81d8-aaf53f544e09)

![MyVA Healthcare no claims.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/df05c92e-9d85-4eeb-b172-ea7496187011)

![My VA User claims, no healthcare.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/fc933101-c8c8-46d3-809f-556ee482832a)

**Comment by Samara-Strauss** [comment]
I see everything here that we talked about, so I don't know that there's anything additional I have to say about the content that we haven't already talked about.

Though I know it's placeholder, here are my thoughts on a right rail — if we decide to have one, it should have a clear goal and purpose. Right now, I'm not sure what its purpose is given that the disability rating and additional links don't feel related. I'm definitely not anti-right rail, so please don't take this comment as that. I just want to make sure that if we have one, we know how we're using it, we feel it's effective, and it's not a dumping ground because we're not sure how to otherwise fit in information. 

Also, this is very small, but I think each of the FTUX blocks should have a title (Health care/Disability compensation/Education) to make those blocks and that experience easier to scan. 

Otherwise, the content is what we talked about, so I think you're ready to move on to the visual design — first, getting the layout more realistically refined and defined (as you mentioned in Slack), and then adding color or other visual details (like some of the established iconography) on top. Really excited to see where this goes!

**Comment by tressaellen** [comment]
@Samara-Strauss - Here's where @callen2563 and I landed yesterday as far as a first pass for overall layout. We're still working on desktop, it's not there yet - but we just wanted to give you an idea of the direction we're thinking of. We're toying with the idea of correlating all the additional links to a specific section. It felt unnatural to have them all clumped together at the bottom. Anyway, I'm excited to discuss things today. Thank you!

![myVA mobile 1.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/76ce05ba-fa44-4da8-8fb3-2f6ec4ec52af)

![My VA desktop 1.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/beb5bcbb-280e-4178-a6c9-88d1db06c8b8)

**Comment by Samara-Strauss** [comment]
We'll discuss more when we meet, but just so you're not waiting hours for feedback, here are some initial thoughts:

- Overall, this looks a lot more streamlined than some of the initial frames we were working on before break, so that's great. It's also way better than what we have now, even with some improvements that could be made.
- I think the general hierarchy looks good.
- On desktop, let's try to not just stretch things and make them really big. If you need to experiment with desktop first and then go back and adjust the mobile view, do that. I'm all for mobile first, but if that means we just stretch things on desktop, we're not doing it right. It's ok to go back and forth and finesse desktop based on mobile and vice versa until you get to the right place.
- With that in mind, on desktop, the disability rating doesn't need that much space. Also, did you only include that note because you felt like you needed something in the right rail? It is unclear to me if veterans are confused enough about outstanding claims not being included in the disability rating that we need to make that kind of clarification on the dashboard (vs. the disability rating section itself).
- On desktop, the additional links blocks don't need to be as long as the height of the section if there's only 1 - 2 links in them.
- Why did you pair the download VA letters links with the claims and appeals information? I am curious about this decision.
- Why did you pair the Post-9/11 GI Bill link with the benefit applications section? I am curious about this decision.

**Comment by Samara-Strauss** [comment]
One more thing re: disability rating:

- Should it go in the name tag? This is an open question — that may or may not be a good idea. This needs design thinking about the goal of the name tag and what it should/should not do.
- Should the disability rating be handled distinctly/differently than the structure that is set up for the rest of the page? It's taking up ton of real estate right now, and it looks like the other "update" content, but it's not. How can we distinguish the difference?

**Comment by Samara-Strauss** [comment]
Not related to the wireframes but the documentation we have on different states: there should probably be a state for claims and appeals where we tell you there are no updates in the last 30 days on (any of) your claim(s). The logic would work like this:

- If someone has any active claims or appeals, show the claims & appeals section.
- If there have been no updates in the last 30 days, show a message saying as much.
- Still show the same CTA to view all claims and appeals.

Also, I think we *should* show the count of how many claims/appointments/prescriptions even if it is only 1. I think in our data schema, we've removed that bit of information if there's only 1, but I don't think we need to. I think it's helpful for clarification from a UX perspective, with the added bonus that it'll make the designed states more consistent.

**Comment by tressaellen** [comment]
@Samara-Strauss - I agree, we can add that into our documentation.

Also (from our previous meeting) this is the direction that we have decided on for the MyVA dashboard.

![My VA desktop 1.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/b0f7d634-ca90-41b7-9a97-632b841835eb)

**Comment by Samara-Strauss** [comment]
I closed this *slightly* prematurely — @tressaellen can you drop in a wireframe here that shows the blank states for each section? Don't worry about the details around if a section will or won't show right now. I just want to have a wireframe that shows what a section would look like if we are showing it but there are no updates. Then, we can re-close.

**Comment by tressaellen** [comment]
@Samara-Strauss here is a mock with all the blank states. I also realized that we didn't have any documentation in the 'Applying for benefits' section for a blank state. I added that with what I have shown in this mock. Do you think that if they don't have any applications in progress that we just don't show that whole section? Or leave it as is with similar messaging to the other sections?

I went with the similar messaging because that would leave the 'What benefits does the VA offer?' dropdown, which I think would be valuable to a user.

![My VA desktop no updates.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/5483e4af-e948-46d9-b894-2663b94e8865)

**Comment by Samara-Strauss** [comment]
> Do you think that if they don't have any applications in progress that we just don't show that whole section? Or leave it as is with similar messaging to the other sections?
> 
> I went with the similar messaging because that would leave the 'What benefits does the VA offer?' dropdown, which I think would be valuable to a user.

I think it's OK to keep this even if someone doesn't have any benefit applications. People can apply for things at any time, plus it's a good prompt and avoids a use care where this page could ever be entirely blank.

And thank you for this mockup! Closing!
